packaging/package_info.rst

```diff
@@ -0,0 +1,190 @@
+Package identifiers and information
+====================================
+
+Packages provide two types of information for consumers. The first one would be the package
+identifiers (ID), which is a SHA-1 hash of the package configuration (settings, options, requirements),
+that allows consumers to reuse an existing package without building it again from sources.
+
+The other type of information would be C/C++ build information, as include paths, library names, or
+compile flags
+
+
+Package IDs
+------------
+
+package_id
```

Comment:
Nop, the ``conanfile_py.rst`` if you meant that, is just for package consumers. This is only relevant for package creators, and I think it should go inside the "Creating packages section".